bundler/spec/other/cli_man_pages_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+RSpec.describe "bundle commands" do
+  it "expects all commands to have a man page" do
+    command_names =
+      Dir["#{source_root}/lib/bundler/cli/*.rb"].
+        grep_v(/common.rb/).
+        map {|file_path| File.basename(file_path, ".rb") }
+
+    command_names.each do |command_name|
+      man_page = "#{source_root}/lib/bundler/man/bundle-#{command_name}.1.ronn"
+      expect(File.exist?(man_page)).to eq true
```

Comment:
With the following tweak, you'll get a better error on failure:

```suggestion
      man_page = source_root.join("lib/bundler/man/bundle-#{command_name}.1.ronn")
      expect(man_page).to exist
```

Instead of something like

```
Failure/Error: expect(File.exist?(man_page)).to eq true
     
expected: true
     got: false
```

you'll get something like

```
Failure/Error: expect(man_page).to exist
  expected #<Pathname:/path/to/rubygems/bundler/lib/bundler/man/bundle-issue.1.ronn> to exist
```